### ISSUE TITLE
Breaking issue due to @react-native-community/cli@4.14.0

### ISSUE THREAD

**Issue by noviadi** [nrwl/nx-react-native#48]
### Breaking issue
Generating a new RN App using `nx-react-native` will use `react-native@0.63.4` that resolves `@react-native-community/cli` dependency to version `4.14.0` which bumps `metro-resolver` to version `0.59` [(changelog)](https://github.com/react-native-community/cli/releases/tag/v4.14.0). This update causes the generated RN App bundler unable to resolve static assets like images or json files.

This example was taken from `nx-react-native` boilerplate:
<img width="844" alt="Screen Shot 2021-02-24 at 17 17 57" src="https://user-images.githubusercontent.com/778268/108986046-4e0bdd00-76c4-11eb-8aff-4091bd23f2aa.png">

**Comment by jaysoo** [comment]
I think the error is  caused by a mismatch between metro-resolver versions.

This PR should fix this https://github.com/nrwl/nx-react-native/pull/50

**Comment by noviadi** [comment]
Thank you for the fix. When could we expect it to be released?

**Comment by jaysoo** [comment]
I'm working on the release today.

**Comment by jaysoo** [comment]
In the meantime, if you install the following packages it should fix it in your repo.

```
metro@0.59.0
metro-resolver@0.59.0
@react-native-community/cli@4.14.0
react-native@0.63.4
```

**Comment by jaysoo** [comment]
This should be fixed now in 11.4.0.

Try running the migration:

```
nx migrate @nrwl/react-native@latest @nrwl/workspace@latest
yarn # or npm install
nx migrate --run-migrations=migrations.json
```
